tormenta-avro/src/main/scala/com/twitter/tormenta/scheme/avro/BufferableScheme.scala

```diff
@@ -16,22 +16,20 @@
 
 package com.twitter.tormenta.scheme.avro
 
-import com.twitter.bijection.Injection
+import com.twitter.bijection.{ Bufferable, Injection }
 import scala.util.{ Failure, Success }
 import com.twitter.tormenta.scheme.Scheme
 import java.nio.ByteBuffer
 
-/**
- * @author Mansur Ashraf
- * @since 9/14/13
- */
-trait AvroScheme[T] extends Scheme[T] {
-  def decodeRecord(buffer: ByteBuffer)(implicit inj: Injection[T, Array[Byte]]): TraversableOnce[T] = {
-    val bytes = Array.ofDim[Byte](buffer.remaining)
-    buffer.get(bytes)
+object BufferableScheme {
```

Comment:
this does not seem to have anything to do with Avro, does it? Why can't this go in the core of tormenta?